### DISCUSSION TITLE
Installing multiple versions?

### DISCUSSION THREAD

**Discussion by FlaMtnBkr** [OrcaSlicer/OrcaSlicer#5323] [General]
I'm sure I will quickly show how little I know, but...

Can multiple versions be installed on the same Windows 10 machine? I thought this was fine and that at one point I had multiple versions installed, but I'm having problems.

I have V1.7.0 "installed" and I'm trying to put V2.0.0 on but if I try to use one then close it and open the other, it wants to "install" it again and the Windows Store warning pops up and it wants to download the "network installer" (which I don't remember exactly what it was called but can find out if important). So it's not allowing both versions to be "installed" at the same time and have to go thru the installation process if I want to switch to the other one. Is this normal?

Also, I'm pretty sure both versions are the portable version. I guess I don't know what that means as I thought that "portable software" meant it could be run without installing on the machine yet both versions are going thru some type of installation process. If I need to grab screen shots I can or if what I have described sounds normal then I will accept it and move on. 

Thanks for anyone that can answer this basic question!

**Comment by vgdh** [comment]
I only use portable version of ORCA because I often test different versions of software and test some new pull requests.
